psignifit/_parameter.py

```diff
@@ -77,6 +77,10 @@ def masked_parameter_bounds(grid: Dict[str, Optional[np.ndarray]], mesh_mask: np
     new_bounds = dict()
     mask_indices = mesh_mask.nonzero()
     for axis, (parameter_name, parameter_values) in enumerate(sorted(grid.items())):
+        if parameter_values is None:
+            new_bounds[parameter_name] = None
+            continue
```

Comment:
are these checks for the benefit of the Matlab tests alone? in this case, we need a different solution

(the same is true for all other similar checks)

Reply:
No, they handle special cases where no bounds are provided.

Reply:
Now after a refactor of fixed_parameters, bounds should never be none. Removed this.